src/test-utils/mocks/mockView.ts

```diff
@@ -0,0 +1,26 @@
+import { mockObject } from 'test-utils/mocks';
+import { ZetkinView } from 'types/zetkin';
+
+const view: ZetkinView = {
+    created: '2021-12-16T14:00:00+00:00',
+    description: 'A random description of this random view',
+    id: 1,
+    organization: {
+        id: 1,
+        title: 'A ',
+    },
+    owner: {
+        id: 1,
+        name: 'M. E. Retsrof',
+    },
+    title: 'A view with a room',
```

Comment:
❤️

Reply:
It's fun @lowlandjuju does literary references in tests and that I do Seinfeld references

Reply:
And I understand neither!